### ISSUE TITLE
Build failure

### ISSUE THREAD

**Issue by J0nnyMak0** [swaywm/sway#6722]
After version change to Wlroots, Sway fails to build:
```
wlroots| Exception: Subproject wlroots version is 0.16.0 but ['>=0.15.0', '<0.16.0'] required.

Subproject subprojects/wlroots is buildable: NO (disabling)
Dependency wlroots found: NO found 0.16.0 but need: '>=0.15.0', '<0.16.0' (overridden)

meson.build:74:1: ERROR: Dependency 'wlroots' is required but not found.
```

**Comment by emersion** [comment]
Fix in https://github.com/swaywm/sway/pull/6721

**Comment by shoober420** [comment]
Same issue here

Build log: [6UUZNC9Y7.txt](https://github.com/swaywm/sway/files/7737876/6UUZNC9Y7.txt)

**Comment by vyivel** [comment]
Fixed by #6721.
